IPython/utils/_get_terminal_size.py

```diff
@@ -0,0 +1,103 @@
+# vendored version of backports.get_terminal_size as nemesapece package are a
+# mess and break, especially on ubuntu. This file is under MIT Licence. 
```

Comment:
It should therefore include the copyright notice attributing the original file. And, technically, the permissions part of the MIT license as well, although I'm not going to complain if we just say 'MIT licence'.

The relevant bit of the MIT licence says:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.